.github/workflows/e2e-nm-angular-workflow.yml

```diff
@@ -44,13 +44,10 @@ jobs:
       run: |
         source scripts/e2e-setup-ci.sh nm
 
-        # TODO: Angular should be fixed to detect the correct package manger to install with
-        # but for now skip the install and do it manually
-        yarn dlx -p @angular/cli@next ng new berry-angular --interactive=false --skip-install
```

Comment:
Could you update the comment instead of removing it?
`@angular/cli` should be updated to correctly detect which package manager to use.

Reply:
done